src/database/postgres.rs

```diff
@@ -335,7 +335,9 @@ impl Postgres {
                         .collect::<Vec<_>>(),
                 ),
                 VisitValue::Value(AbiValue::Bytes(v)) => Box::new(v.to_owned()),
-                VisitValue::Value(AbiValue::String(v)) => Box::new(v.to_string()),
+                VisitValue::Value(AbiValue::String(v)) => {
+                    Box::new(v.trim_matches('\0').to_string())
```

Comment:
Are we removing leading null characters intentionally here?

Reply:
Technically we are, although I don't think there won't be null leading characters because of the way the hex strings are processed. We can not insert these null characters into the DB so we need to remove them all anyway.